index.js

```diff
@@ -39,6 +40,7 @@ app.use('/customer', passport.authenticate('jwt-customer', {session: false}),cus
 app.use('/rider', passport.authenticate('jwt-rider', {session: false}),riderRouter);
 app.use('/staff', passport.authenticate('jwt-staff', {session: false}),staffRouter);
 app.use('/manager', passport.authenticate('jwt-manager', {session: false}),managerRouter);
+app.use('/orders', passport.authenticate('jwt-manager', {session: false}), ordersRouter);
```

Comment:
You should add manager related routes to /manager router. I think it would be good to use the empty router/manager.js. If its too much code for 1 file, u can add another router inside router/manager.js instead of adding to the main index.js